lib/c.nit

```diff
@@ -138,6 +138,18 @@ class CByteArray
 		end
 		return carray
 	end
+
+	# Safely move `n` bytes from `dst_offset` to `src_offset`, inside this array
+	#
+	# Require: all arguments greater than 0 and ranges within `length`
+	fun memmove(dst_offset, src_offset, n: Int)
```

Comment:
I'm a bit surprised with the `memmove`, if you decide to go with C's name, I think it would be wise to have the same signature as the original function.

Maybe rename it as `move` ?